docs/concepts/components/callbacks.md

```diff
@@ -92,3 +92,7 @@ html! {
     <input type="text" onkeypress=onkeypress />
 }
 ```
+
+## Relevant Examples
```

Comment:
I think generally with capitalisation of titles, we've been going for first word capitalised, followed by all other words (sans proper nouns, proper adjectives, etc) lowercase.
This is a really minor nit though.
```suggestion
## Relevant examples
```